invenio_records_resources/proxies.py

```diff
@@ -21,3 +31,7 @@
     lambda: current_app.extensions["invenio-records-resources"].notification_registry
 )
 """Helper proxy to get the current notifications registry."""
+
+current_transfer_registry: "TransferRegistry" = LocalProxy(
```

Comment:
normally invenio does not add type checks. to be consistent with the rest of invenio i would remove `: "TransferRegistry"`

Reply:
and i saw there are a lot of type hints in other files too